.github/workflows/main.yml

```diff
@@ -34,6 +34,11 @@ jobs:
         uses: actions/checkout@v2.3.3
         with:
           submodules: 'recursive'
+      # This will probably not be necessary once GitHub actions
+      # uses an Xcode that supports macOS arm64 by default.
```

Comment:
In fact, we should probably just sit on this since it looks like this will be automatically fixed on Nov 30: https://github.com/actions/virtual-environments/issues/2056

Reply:
@johnboiles Small reminder to remove this if it is truly not required anymore by tomorrow.